LoopKitUI/OnboardingUI.swift

```diff
@@ -132,8 +132,16 @@ public protocol TherapySettingsProvider {
     var onboardingTherapySettings: TherapySettings { get }
 }
 
+public enum StudyProduct: String, CaseIterable {
+    case none
+    case studyProduct1
+    case studyProduct2
+}
```

Comment:
I'm not against this being here, but it seems that this is specific to Tidepool Loop and doesn't need to be in LoopKit. Do the device managers need access to this?

Reply:
@nhamming it's used below as part of the OnboardingProvider protocol -- similar to `allowDebugFeatures`

Reply:
yeah, but use of `allowDebugFeatures` seems right for all device managers and services, but study products seems only for Tidepool related studies.

Reply:
Ahh I see. Would you suggest I create a new protocol in `TidepoolOnboarding` then with this single requirement?

Reply:
Moved